backend/src/v2/driver/util.go

```diff
@@ -76,3 +80,95 @@ func resolvePodSpecInputRuntimeParameter(parameterValue string, executorInput *p
 	}
 	return parameterValue, nil
 }
+
+// resolveK8sParameter resolves a k8s JSON and unmarshal it
+// to the provided k8s resource.
+//
+// Parameters:
+//   - pipelineInputParamSpec: An input parameter spec that resolve to a valid structpb.Value
+//   - inputParams: InputParams that contain resolution context for pipelineInputParamSpec
+func resolveK8sParameter(
```

Comment:
What's the purpose of this function? It seems like it's a wrapper around `resolveInputParameter` but doesn't actually do anything but return a generic error format.

Reply:
good catch, this should have been removed after we moved to share protos